src/transformers/models/fast/modeling_fast.py

```diff
@@ -0,0 +1,380 @@
+    searched for image classification.""",
+    FAST_START_DOCSTRING,
+)
+class FastForSceneTextRecognition(FastPreTrainedModel):
+    def __init__(self, config):
+        super().__init__(config)
+        self.config = config
+
+        if config.use_timm_backbone:
+            requires_backends(self, ["timm"])
+            kwargs = {}
+            if config.dilation:
+                kwargs["output_stride"] = 16
+            backbone = create_model(
+                config.backbone,
+                pretrained=config.use_pretrained_backbone,
+                features_only=True,
+                out_indices=(1, 2, 3, 4),
+                in_chans=config.num_channels,
+                **kwargs,
+            )
+        else:
+            backbone = load_backbone(config.backbone_config)
+        self.backbone = backbone
+        self.neck = FASTNeck(config=config)
+        self.text_detection_head = FASTHead(config=config)
+
+        self.pooling_1s = nn.MaxPool2d(
+            kernel_size=config.head_pooling_size, stride=1, padding=(config.head_pooling_size - 1) // 2
+        )
+        self.pooling_2s = nn.MaxPool2d(
+            kernel_size=config.head_pooling_size // 2 + 1, stride=1, padding=(config.head_pooling_size // 2) // 2
+        )
+        self.post_init()
+
+    @add_start_docstrings_to_model_forward(FAST_FOR_CAPTIONING_INPUTS_DOCSTRING)
+    @replace_return_docstrings(output_type=FastForSceneTextRecognitionOutput, config_class=_CONFIG_FOR_DOC)
+    def forward(
+        self,
+        pixel_values: torch.FloatTensor,
+        output_hidden_states: Optional[bool] = True,
+        return_dict: Optional[bool] = None,
+        labels: Dict = None,
+    ):
+        r"""
+        labels (`Dict[str, torch.Tensor]`, *optional*):
+            Should contain 3 keys: gt_texts,gt_kernels,gt_instances
+
+        Returns:
+
+                Examples:
+
+        ```python
+        >>> from transformers import FastImageProcessor, FastForSceneTextRecognition
+        >>> from PIL import Image
+        >>> import requests
+
+        >>> url = "https://huggingface.co/datasets/Raghavan/fast_model_samples/resolve/main/img657.jpg"
+        >>> image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+        >>> processor = FastImageProcessor.from_pretrained("jadechoghari/fast-tiny")
+        >>> model = FastForSceneTextRecognition.from_pretrained("jadechoghari/fast-tiny")
+        >>> inputs = processor(image, return_tensors="pt")
+        >>> # forward pass
+        >>> outputs = model(pixel_values=inputs["pixel_values"])
+        >>> target_sizes = [(image.height, image.width)]
+        >>> threshold = 0.88
+        >>> text_locations = processor.post_process_text_detection(outputs, target_sizes, threshold, bbox_type="rect")
+        >>> print(text_locations[0]["bboxes"][0][:10])
+        [151, 151, 160, 56, 355, 74, 346, 169]
```

Comment:
```suggestion
        >>> from transformers import FastImageProcessor, FastForSceneTextRecognition
        >>> from PIL import Image
        >>> import requests

        >>> url = "https://huggingface.co/datasets/Raghavan/fast_model_samples/resolve/main/img657.jpg"
        >>> image = Image.open(requests.get(url, stream=True).raw).convert("RGB")

        >>> processor = FastImageProcessor.from_pretrained("jadechoghari/fast-tiny")
        >>> model = FastForSceneTextRecognition.from_pretrained("jadechoghari/fast-tiny")

        >>> inputs = processor(image, return_tensors="pt")
        >>> with torch.no_grads():
        ...     outputs = model(pixel_values=inputs["pixel_values"])

        >>> target_sizes = [(image.height, image.width)]
        >>> text_locations = processor.post_process_text_detection(outputs, target_sizes, threshold=0.88, bbox_type="rect")
        >>> print(text_locations[0]["bboxes"][0][:10])
        [151, 151, 160, 56, 355, 74, 346, 169]
```
Boxes should be in `center_x, center_y, width, height, angle` format (in case we have angle), otherwise I'm not sure I got what is bboxes here?

Reply:
but yeah it's gonna follow the convention: `[(x1, y1), (x2, y2), (x3, y3), (x4, y4)]`